packages/rpc-graphql/src/schema/root.ts

```diff
@@ -2,6 +2,7 @@ export const rootTypeDefs = /* GraphQL */ `
     type Query {
         account(address: Address!, commitment: Commitment, minContextSlot: Slot): Account
         block(slot: Slot!, commitment: CommitmentWithoutProcessed): Block
+        multipleAccounts(addresses: [Address!]!, commitment: Commitment, minContextSlot: Slot): [Account]
```

Comment:
Usually just follow pluralization rules.
```suggestion
        accounts(addresses: [Address!]!, commitment: Commitment, minContextSlot: Slot): [Account]
```